test/e2e/service.go

```diff
@@ -797,6 +797,75 @@ var _ = Describe("Services", func() {
 	})
 })
 
+var _ = Describe("Services", func() {
+	f := NewFramework("services-temp")
+
+	var t *explorerTest
+	BeforeEach(func() {
+		t = newDefaultExplorerTest(f.Client, f.Namespace.Name)
+	})
+
+	It("should serve round-robin services round-robin", func() {
```

Comment:
This does test round robin for 2 node clusters because we check that the lastHostname != currentHostname but that doesn't work  for >2node clusters. I will look into revising this.